### DISCUSSION TITLE
Remove "v" from version badge?

### DISCUSSION THREAD

**Discussion by pokemon973** [badges/shields#6135] [Q&A] [answered]
:clipboard: **Description**
Hello,
Is there a way to remove "v" from badge??
Example. 2.0 not v2.0

Thanks!!

**Comment by calebcartwright** [comment, marked as answer]
Thanks for reaching out with your question! It's not clear if your question is in reference to a specific version badge or in general, but no this is not possible directly today.

One of our core goals is consistency across our badges (it's a big part of why the service exists in the first place!) which means that there's various conventions applied across all badges, including some category-specific conventions (build, license, version, etc.). One such long established and consistently applied convention is the inclusion of the `v` prefix on the badge message for version badges.

We're aware that there are some folks that would like to elide the `v` (we've had maybe 2 or 3 reports/questions over the last several years), but we couldn't change that behavior after such a long time, and we're highly unlikely to accept any new customization options.

We do think it's important that the badges be configurable to a degree, which is why we offer different styles, colors, and the ability to customize text like the label. However, we don't have a goal of being infinitely configurable, and the line is typically drawn at the value of the badge message. There's a lot of reasons for this, notably because doing so would almost certainly snowball into myriad customization requests and divergent badges, which quickly starts running counter to our goals, as well as the increased complexity and maintenance burden on the code itself.

However, for anyone that feels strongly about needing a badge message to look differently, we'd recommend trying the [Endpoint badge](https://shields.io/endpoint). This can be leveraged to have ultimate flexibility of your badges, and does so without the negative drawbacks on Shields.io itself.

Such a custom endpoint for someone that wanted to drop the the `v` from the badge message could continue to leverage Shields to do the work by requesting json from us (e.g. your endpoint could fetch `https://img.shields.io/gem/v/formatador.json`), manipulating the message as you see fit, and then returning a response in the Endpoint form. Then in your markdown files (or wherever you are displaying your badges) you could simply use your endpoint badge url instead!
